src/Utils/AuthorizeFor.tsx

```diff
@@ -27,6 +27,10 @@ export type AuthorizedElementProps = {
   authorizeFor?: AuthorizedForCB | undefined;
 };
 
+export const NonReadOnlyUsersList = USER_TYPES.filter((userType) => {
+  return !userType.includes("ReadOnly");
+});
```

Comment:
We already have the filter function defined just below it. Re-use it.

```suggestion
export const NonReadOnlyUsersList = USER_TYPES.filter(NonReadOnlyUsers);
```